src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import "./Introduce.scss";
+import { useNavigate } from "react-router-dom";
+
+const IntroduceName = () => {
+  const navigate = useNavigate();
+
+  const logoname = "> wecode";
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+  const handleButton = (e) => {
+    e.preventDefault();
+    navigate("/product");
+    console.log("button 태그 동작");
+  };
```

Comment:
```suggestion
  const handleInput = () => {
    console.log("input 태그 동작");
  };
  
  const handleButton = (e) => {
    e.preventDefault();
    navigate("/product");
    console.log("button 태그 동작");
  };
```
함수끼리 가독성을 위해서 한칸씩 띄워주세요!

Reply:
넵!